commands/command_checkout.go

```diff
@@ -15,6 +15,7 @@ var (
 	checkoutCmd = &cobra.Command{
 		Use:   "checkout",
 		Short: "Checks out LFS files into the working copy",
+		Long:  git_lfs_checkout_HelpText,
```

Comment:
Go keywords tend to be in camelCase. Also, I think the `git_lfs` prefix here is redundant. I think you could name these after the subcommand: `checkoutHelp`, `cleanHelp`, etc.